test/test_utils/ec2.py

```diff
@@ -911,7 +911,42 @@ def post_process_mxnet_ec2_performance(connection, log_location):
         raise ValueError("total: {}; n: {} -- something went wrong".format(total, n))
 
 
-@retry(reraise=True, wait=wait_fixed(60), stop=stop_after_attempt(5))
```

Comment:
Why drop the retries?

Reply:
The `kill` function is a deterministic solution to the problem of the background process preventing us from running `apt-get update` ourselves.

So there is no more need to retry this function in the hope that it will work on the 2nd-5th attempts. It should always work in the first attempt.